frontend/src/services/http/http.service.ts

```diff
@@ -40,7 +40,7 @@ class Http {
     return response;
   }
 
-  _parseJSON<T>(response: Response): Promise<T> {
+  private _parseJSON<T>(response: Response): Promise<T> {
     return response.json();
   }
 
```

Comment:
The Http class also has a _throwError method, perhaps it should also have a private access modifier

Reply:
good catch 👍

Reply:
Thanks)